benchparse.py

```diff
@@ -19,16 +19,22 @@
 parameters.
```

Comment:
I couldn't decide. It didn't quite feel like it belonged there, but I could go either way.